docs/src/main/tut/docs.md

```diff
@@ -1184,8 +1161,6 @@ Note that Cats' typeclass hierarchy is expressed with inheritance and methods fr
 We make use of the `FromMonadError` class below, making it easer to implement `FetchMonadError[Task]` given a `MonadError[Task, Throwable]` which we can get from the _monix-cats_ projects.
 
 ```tut:silent
-import monix.cats._
-
 implicit val taskFetchMonadError: FetchMonadError[Task] =
   new FetchMonadError.FromMonadError[Task] {
     override def runQuery[A](q: Query[A]): Task[A] = queryToTask[A](q)
```

Comment:
Has this method been removed [here](https://github.com/47deg/fetch/pull/137/files?diff=split#diff-6d6feb92c529f59043fb943fd182a5e2L30)? Should this be updated?

Reply:
I removed the method from `FromMonadError` because it is inherited from `FetchMonadError`.